# Defines how fast new measurements are averaged into the transformation estimate (new_value = (1-update_rate)*old_value + update_rate*measurement). [0,1]
# double
update_rate: 0.25

# The name of the computed child frame.
# string
child_frame_name: "/landmark_reference_nav"

# the (minimum) length of the frontal wall segment left of the laser scanner's origin (assuming the robot faces the frontal wall), in [m]
# double
wall_length_left: 0.3

# the (minimum) length of the frontal wall segment right of the laser scanner's origin (assuming the robot faces the frontal wall), in [m]
# double
wall_length_right: 0.3

# the maximum distance of the side wall to the laser scanner, in [m]
# double
max_wall_side_distance: 3.0
